### PR TITLE
Google Colab integration

### PR DESCRIPTION
**Pull request by rvlb**

### CODE REVIEW

**Review by fjsj — CHANGES_REQUESTED** [review]
- Could you please make a Colab for notebooks/End-to-End-Matching-Example.ipynb?
- Please also check all links inside the Colab notebooks.

**Comment by fjsj** [comment]
Also, in the examples section in readme, please make a table like this with colab links: https://github.com/KevinMusgrave/pytorch-metric-learning/blob/master/examples/README.md

**Comment by fjsj** [comment]
One more thing: is there a way to force Colab to use a GPU? I had to manually use "Change runtime type" to use a GPU.

**Comment by rvlb** [comment]
Not that I've found, pytorch-metric-learning also asks that we do it manually

Our readme: https://github.com/vintasoftware/entity-embed/blob/5f3b34e8b42bbed4f621a1c5a37edf7de75e5509/notebooks/google-colab/README.md

**Comment by fjsj** [comment]
- Please keep the current descriptions we have in README for the examples.
- Make a Colab for notebooks/End-to-End-Matching-Example.ipynb. Not sure how to load the model though, maybe we'll have to pre-generate it. Please research if possible to have disk persistence in Colab across Notebooks.
- If it's not possible to programatically request a GPU, then please add the instruction to the top of the Colab Notebook.

**Comment by rvlb** [comment]
Persistence seems to be possible via Google Drive integration https://stackoverflow.com/a/66670818

**Comment by fjsj** [comment]
> Persistence seems to be possible via Google Drive integration

In this case, we can leave the code that saves to Drive commented and put an explanation on the Notebook.

File: notebooks/google-colab/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Google Colab Examples
+
+Before running the notebooks on Google Colab, please make sure to set the runtime type to "GPU". Do that by going to the "Runtime" menu and selecting "Change runtime type".
+
+## Examples
+
+Notebook | Description | Google Colab Link
+--- | --- | ---
+Deduplication Example | For when you have a single dirty dataset with duplicates | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/vintasoftware/entity-embed/blob/main/notebooks/google-colab/Deduplication-Example.ipynb)
```
Comment (rvlb) [inline notebooks/google-colab/README.md:9]: Please note that these urls consider the files already exist on main.

**Review by fjsj — APPROVED** [review]
LGTM

**Pull request merged**